src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -1311,7 +1311,8 @@ private void Initialized(object message)
             if (message is GossipEnvelope)
             {
                 var ge = message as GossipEnvelope;
-                ReceiveGossip(ge);
+                var receivedType = ReceiveGossip(ge);
+                _log.Debug("Cluster Node [{0}] - Received gossip from [{1}] which was {2}.", _cluster.SelfAddress, ge.From, receivedType);
```

Comment:
in scenarios where gossip looks like it isn't being processed properly, being able to determine what the non-leader nodes did with it is key. This information should help especially when combined with https://github.com/akkadotnet/akka.net/pull/2791